Sources/MongoDBVapor/Exports.swift

```diff
@@ -1 +1,2 @@
-@_exported import func MongoSwift.cleanupMongoSwift
+// re-export the driver
+@_exported import MongoSwift
```

Comment:
I keep forgetting that re-exporting is only hard when you only want to re-export select symbols 🙂

Reply:
Yep, I realized this too when doing the SwiftBSON stuff in the driver. I originally had an `@_exported` line for each symbol until I found out it wasn't necessary...